frontend/src/components/MealPlanning/AddMealPlan.jsx

```
@@ -136,15 +138,15 @@ const PlanController = ({ startDate, currentDate, toggleReload }) => {
       title: "Meal Plan is being saved. Please wait..",
       type: "loading",
     });
-    setFiltersApplied(true);
+    setPlanAdded(true);
     let newPlan = { ...plan };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'newPlan' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)